.github/workflows/gh-pages-releases.yml

```diff
@@ -0,0 +1,40 @@
+# See: https://github.com/llvm/torch-mlir/issues/1374
+name: Publish releases page
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+  workflow_dispatch:
+
+jobs:
+  scrape_and_publish_releases:
+    name: "Scrape and publish releases"
+    runs-on: ubuntu-20.04
+
+    # Don't run this in everyone's forks.
+    if: github.repository == 'llvm/torch-mlir'
+
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WORKFLOW_INVOCATION_TOKEN }}
+      - name: Run scrape releases script
+        run: python ./tools/scrape_releases.py llvm torch-mlir > /tmp/index.html
```

Comment:
nit: let's put this in build_tools since it is a "meta" tool and "not part of the compiler"

Reply:
Done.